composer.json

```diff
@@ -1,21 +1,13 @@
 {
-    "name": "refactoredgroup/autoffl-m2-module",
+    "name": "refactored-group/automatic-ffl-magento",
     "description": "Automatic FFL Magento 2 Extension",
     "replace": {
         "refactoredgroup/autoffl-core": "self.version",
```

Comment:
I see that the parent package was called `refactored-group/` but the child packages are `refactoredgroup/`. Can we make them all match what Github says (`refactored-group/`)